wasmbinding/test/custom_query_test.go

```diff
@@ -116,7 +116,7 @@ func (suite *CustomQuerierTestSuite) TestInterchainQueryResultNotFound() {
 	}
 	resp := icqtypes.QueryRegisteredQueryResultResponse{}
 	err := suite.queryCustom(ctx, contractAddress, query, &resp)
-	expectedErrMsg := "Generic error: Querier contract error: codespace: interchainqueries, code: 1115: query wasm contract failed"
+	expectedErrMsg := "Generic error: Querier contract error: codespace: interchainqueries, code: 1116: query wasm contract failed"
```

Comment:
```suggestion
	expectedErrMsg := fmt.Sprintf("Generic error: Querier contract error: codespace: interchainqueries, code: %d: query wasm contract failed", icqtypes.ErrNoQueryResult.ABCICode())
```
what do you think? we wouldn't have to update this part of code anymore